### PR TITLE
Upgrade factory boy

### LINKED ISSUE #26: FactoryBoy needs upgrade
Currently, running ``py.test mysite`` raises this error:
```
E   ImportError: The ``fake-factory`` package is now called ``Faker``.
E
E   Please update your requirements.
```
This is an issue which has been resolved in FactoryBoy: https://github.com/FactoryBoy/factory_boy/issues/334
Faker changed the way it should be imported and because the version of Faker was loosely specified in FactoryBoy requirements, a build that used to work doesn't work anymore.
I can send a PR with an updated ``requirements.txt`` if that helps.

### PR DESCRIPTION
**Pull request by ghost**
Currently, running `py.test mysite` raises this error:

    E   ImportError: The ``fake-factory`` package is now called ``Faker``.
    E
    E   Please update your requirements.

    This is an issue which has been resolved in FactoryBoy:
    FactoryBoy/factory_boy#334
    Faker changed the way it should be imported and because the version of
    Faker was loosely specified in FactoryBoy requirements, a build that
    used to work doesn't work anymore.

    Fix #26

### CODE REVIEW

**Comment by tisto** [comment]
@amarandon thanks for your pull request! Would you mind adding a changelog entry? Feel free to add yourself to the list of contributors as well...

**Comment by ghost** [comment]
I've updated the changelog. I couldn't find the list of contributors though.

File: CHANGES.rst [hunk 1]
```diff
@@ -7,6 +7,7 @@
 
 - Remove deprecated method from internal startup flow.
 - Use AUTH_MODEL to create user instead of django's default
+- Upgrade FactoryBoy to version 2.8.1
```
Comment (tisto) [inline CHANGES.rst:10]: @amarandon please add your name in brackets below the changelog.

**Comment by tisto** [comment]
@amarandon sorry, my fault. I will create such a list and add your name to it. Again, thanks for contributing!

**Pull request merged**